reposerver/repository/repository.go

```diff
@@ -1645,3 +1645,58 @@ func (s *Service) TestRepository(ctx context.Context, q *apiclient.TestRepositor
 	}
 	return apiResp, nil
 }
+
+func (s *Service) ResolveRevision(ctx context.Context, q *apiclient.ResolveRevisionRequest) (*apiclient.ResolveRevisionResponse, error) {
```

Comment:
Since this is a public method, can you please give it some descriptive commentary?

Reply:
Sure, I will add method description on this.